pyproject.toml

```diff
@@ -16,6 +15,9 @@ max-line-length = 119
 # Based on Google Python Style Guide
 disable = [
     "C0301",  # Line too long (handled by black)
+    "C0114",  # Missing module docstring (missing-module-docstring)
+    # This repo contains examples that should remain independent, so we expect some level of copy/paste
+    "R0801",  # Similar lines in 2 files
```

Comment:
As the comment says, we expect some level of copy paste between the examples. So ignoring this warning.